test/common/config/datasource_test.cc

```diff
@@ -1,15 +1,18 @@
 #include "envoy/config/core/v3/base.pb.h"
+#include "envoy/config/core/v3/base.pb.validate.h"
```

Comment:
Please switch to `TestUtility::loadFromYamlAndValidate` throughout this file to future proof. Also please merge master which should finally fix coverage. Thank you!

/wait

Reply:
Done